plugins/embark-assistant/biome_type.cpp

```diff
@@ -0,0 +1,752 @@
+/* The code is copied from Ragundo's repo referenced below.
+The changes are:
+- The addition of a .h file reference.
+- The simplification of the code using ofsub to remove the use of (and
```

Comment:
Yeah
That's was one artifact from IDA reverse code generation.
I never really understood its purpose, and I see now that is has no effects in the code removing it.
I take note of this and change the code in my code.
Thanks!!!